src/apis/api_v1.py

```
@@ -0,0 +1,220 @@
+        if webhook_config:
+            webhook = CustomWebhook(WebhookConfig(**webhook_config))
+            logger.info(
+                f"Posting results to the webhook configured at {webhook.config.endpoint}"
+            )
+            res = webhook.post_result({"results": results, "session_id": fa.session.id})
+            logger.info(f"Results posted to the webhook with res: {str(res)}")
+
+        return {"result": results, "session_id": session_id}
+    except Exception as err:
+        logger.error(err)
+        raise Exception(str(err))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)